### PR TITLE
Replace vscaleexpminusmax yaml with header

### PR DESCRIPTION
**Pull request by RahulSundarMCW**

### CODE REVIEW

**Review by dsharlet — COMMENTED** [review]
Same comments as #7408

**Comment by RahulSundarMCW** [comment]
Address the review comments and pushed the commit.

File: src/f32-vscaleexpminusmax/f32-vscaleexpminusmax.h [hunk 1]
```diff
@@ -2,51 +2,54 @@
...
-XNN_UKERNEL_WITH_PARAMS(xnn_arch_x86_avx512f, xnn_f32_vscaleexpminusmax_ukernel__avx512f_p5_scalef_u128, 128, float, struct xnn_f32_default_params, ((xnn_f32_vscaleexpminusmax_ukernel_fn) NULL))
-XNN_UKERNEL_WITH_PARAMS(xnn_arch_x86_avx512f, xnn_f32_vscaleexpminusmax_ukernel__avx512f_p5_scalef_u144, 144, float, struct xnn_f32_default_params, ((xnn_f32_vscaleexpminusmax_ukernel_fn) NULL))
-XNN_UKERNEL_WITH_PARAMS(xnn_arch_x86_avx512f, xnn_f32_vscaleexpminusmax_ukernel__avx512f_p5_scalef_u160, 160, float, struct xnn_f32_default_params, ((xnn_f32_vscaleexpminusmax_ukernel_fn) NULL))
-XNN_UKERNEL_WITH_PARAMS(xnn_arch_x86_avx512f, xnn_f32_vscaleexpminusmax_ukernel__avx512f_p5_scalef_u176, 176, float, struct xnn_f32_default_params, ((xnn_f32_vscaleexpminusmax_ukernel_fn) NULL))
-XNN_UKERNEL_WITH_PARAMS(xnn_arch_x86_avx512f, xnn_f32_vscaleexpminusmax_ukernel__avx512f_p5_scalef_u192, 192, float, struct xnn_f32_default_params, ((xnn_f32_vscaleexpminusmax_ukernel_fn) NULL))
+#if XNN_ENABLE_AVX512F_U32 && (XNN_ARCH_X86 || XNN_ARCH_X86_64)
```
Comment (RahulSundarMCW) [inline src/f32-vscaleexpminusmax/f32-vscaleexpminusmax.h:33]: With XNN_ENABLE_AVX512F enabled, we encountered illegal instruction errors in these tests. We replaced it with XNN_ENABLE_AVX512F_U32. Can you look into this.

File: tools/generate-vscaleexpminusmax-test.py
Comment (dsharletg) [inline tools/generate-vscaleexpminusmax-test.py:1]: Delete this file

**Review by dsharlet — APPROVED** [review]

**Review by dsharlet — APPROVED** [review]

**Pull request merged**